src/io/ResultWriter.cpp

```diff
@@ -47,11 +48,15 @@ void ResultWriter::endStep(ParticleContainer * /*particleContainer*/, DomainDeco
 	_U_pot_acc->addEntry(domain->getGlobalUpot());
 	_p_acc->addEntry(domain->getGlobalPressure());
 	if((domainDecomp->getRank() == 0) && (simstep % _writeFrequency == 0)){
-		_resultStream << simstep << "\t" << _simulation.getSimulationTime()
-			<< "\t\t" << domain->getGlobalUpot() << "\t" << _U_pot_acc->getAverage()
-			<< "\t\t" << domain->getGlobalPressure() << "\t" << _p_acc->getAverage()
-			<< "\t\t" << domain->getGlobalBetaTrans() << "\t" << domain->getGlobalBetaRot()
-			<< "\t\t" << domain->cv() << "\t\t" << domain->getglobalNumMolecules();
+		_resultStream << std::setw(10) << simstep << std::setw(28) << std::scientific << std::setprecision(std::numeric_limits<double>::digits10) << _simulation.getSimulationTime()
+			<< std::setw(28) << std::scientific << std::setprecision(std::numeric_limits<double>::digits10) << domain->getGlobalUpot()
+			<< std::setw(28) << std::scientific << std::setprecision(std::numeric_limits<double>::digits10) << _U_pot_acc->getAverage()
+			<< std::setw(28) << std::scientific << std::setprecision(std::numeric_limits<double>::digits10) << domain->getGlobalPressure()
+			<< std::setw(28) << std::scientific << std::setprecision(std::numeric_limits<double>::digits10) << _p_acc->getAverage()
+			<< std::setw(28) << std::scientific << std::setprecision(std::numeric_limits<double>::digits10) << domain->getGlobalBetaTrans()
+			<< std::setw(28) << std::scientific << std::setprecision(std::numeric_limits<double>::digits10) << domain->getGlobalBetaRot()
+			<< std::setw(28) << std::scientific << std::setprecision(std::numeric_limits<double>::digits10) << domain->cv()
+			<< std::setw(28) << std::scientific << std::setprecision(std::numeric_limits<double>::digits10) << domain->getglobalNumMolecules();
```

Comment:
do you need this 10 digit precision here?
6 would suffice and wouldn't break the current validation tests that we have implemented.

Reply:
The main goal was to improve the formatting of the result file by substituting the tabs by using fixed field widths. I just took the precision from the MettDaemon plugin. As suggested, the best would be to make the precision configurable (Default value: 6). I am working on it...